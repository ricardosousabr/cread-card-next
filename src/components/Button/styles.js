import styled from 'styled-components'

export const ButtonAddCard = styled.button`
  width: 200px;
  height: 40px;
  background-color: aqua;
  border: none;
  border-radius: 5px;
`
