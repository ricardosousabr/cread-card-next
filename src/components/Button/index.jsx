import { ButtonAddCard } from './styles'

export default function Button(props) {
  return <ButtonAddCard>{props.children}</ButtonAddCard>
}
