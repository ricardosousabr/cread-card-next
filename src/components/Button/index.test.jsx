/**
 * @jest-environment jsdom
 */

import React from 'react'
import Button from './index'
import renderer from 'react-test-renderer'

test('HomePage snapshot', () => {
  const component = renderer.create(<Button />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
