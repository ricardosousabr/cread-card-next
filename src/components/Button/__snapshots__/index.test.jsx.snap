// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage snapshot 1`] = `
<button
  className="styles__ButtonAddCard-sc-11nj3cj-0 eZHHSY"
/>
`;
